README.md

```diff
@@ -1,7 +1,334 @@
 # Coveo C# Platform SDK
-The offical Coveo C# SDK allows your software to communicate with the Coveo Cloud Platform public APIs and the Push API.
+[![NuGet version (Coveo.Connectors.Utilities.PlatformSdk)](https://img.shields.io/nuget/v/Coveo.Connectors.Utilities.PlatformSdk.svg?style=flat-square)](https://www.nuget.org/packages/Coveo.Connectors.Utilities.PlatformSdk/)
 
-The code of the SDK is not open-source. This repo is meant to provide feedback, bug reporting, etc.
+The official Coveo C# SDK allows your software to communicate with the Coveo Cloud Platform public APIs and the Push API.
+
+The package is .NET Standard 2.0 compatible and therefore works with .NET Core and .NET Framework
+
+The code of the SDK is not open-source. This repository is meant to provide feedback, bug reporting, etc.
```

Comment:
to provide feedback -> to handle feedback

Reply:
```suggestion
The code of the SDK is not open-source. This repository is meant to handle feedback, bug reporting, etc.
```